samples/Mvc.RenderViewToString/project.json

```diff
@@ -1,7 +1,10 @@
 {
   "buildOptions": {
     "emitEntryPoint": true,
-    "preserveCompilationContext": true
+    "preserveCompilationContext": true,
+    "copyToOutput": {
+      "include": "Views"
```

Comment:
We don't set the content root to the current working directory anywhere so this is required to make `dotnet run` work  out of the box.